htdocs/societe/card.php

```diff
@@ -120,7 +120,7 @@
 $socialnetworks = getArrayOfSocialNetworks();
 
 // Initialize technical object to manage hooks of page. Note that conf->hooks_modules contains array of hook context
-$hookmanager->initHooks(array('thirdpartycard', 'globalcard'));
+$hookmanager->initHooks(array('thirdpartycard', 'globalcard', 'customizeprospectcustomerfield'));
```

Comment:
Don't need another context here. The context is the "main page" where the hook will be called.
You already have thirdpartycard